Backend/app/dashboard/CommonAPIs/spec/ProviderPlatform/Management/API/DriverRegistration.yaml

```diff
@@ -139,6 +139,25 @@ apis:
         endpoint: DriverRegistrationAPI UpdateDocumentEndpoint
         endpointV2:
         userActionType: ApiAuth DRIVER_OFFER_BPP_MANAGEMENT DRIVERS UPDATE_DOCUMENT
+  - POST: #UnlinkDocumentAPI
+      endpoint: /{personId}/unlink/document/{documentType}
+      auth: ApiAuthV2
+      params:
+        personId: Id Driver
+        documentType: DocumentType
+      response:
+        type: APISuccess
+      helperApi:
+        - POST:
+            endpoint: /{personId}/unlink/document/{documentType}
+            auth: ApiAuthV2
+            params:
+              personId: Id Driver
+              documentType: DocumentType
+            query:
+              - requestorId: Text
+            response:
+              type: Bool
```

Comment:
Can you add proper type instead of just Bool ?
For example
`newtype UnlinkDocumnentResp = UnlikDocumnentResp {isFleetOwnerUploadedAllMandatoryDocument :: Bool}`